Chatto/Tests/ChatController/ChatMessagesViewControllerTests.swift

```diff
@@ -0,0 +1,372 @@
+//
+// The MIT License (MIT)
+//
+// Copyright (c) 2015-present Badoo Trading Limited.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+import UIKit
+import XCTest
+@testable import Chatto
+
+class ChatMessagesViewControllerTests: XCTestCase {
+
+    func testThat_WhenViewControllerInitated_ThenViewsIsNotLoaded() {
+        let chatMessageTestComponents = ChatMessageTestComponents()
+        let messagesViewController = chatMessageTestComponents.viewController
+
+        XCTAssertFalse(messagesViewController.isViewLoaded)
+        XCTAssertNotNil(messagesViewController.collectionView)
```

Comment:
I think originally there was `XCTAssertNil` here. Should we even check it?

Reply:
Since `ChatMessagesViewController` is a UICollectionViewController subclass we will have this view "for granted" now.
Probably does not make so much sense this check, since we are testing the native class behaviour 👍